examples/ansible-vars-oke.yaml

```diff
@@ -0,0 +1,30 @@
+## Cluster
+PROVIDER: custom
+
+NAMESPACE: viyanamespace
+
+## MISC
+DEPLOY: true # Set to false to stop at generating the manifest
+
+## Storage
+V4_CFG_MANAGE_STORAGE: true
+
+## SAS API Access
+# raw not base64
+V4_CFG_SAS_API_KEY: '<api_client_id>'
+V4_CFG_SAS_API_SECRET: '<api_client_secret>'
+V4_CFG_ORDER_NUMBER:  '<order_number>'
+
+## Ingress
+V4_CFG_INGRESS_TYPE: ingress
+V4_CFG_INGRESS_FQDN: 'viya.publb.oke.internal'
+V4_CFG_TLS_MODE: "full-stack" # [full-stack|front-door|disabled]
+
+
```

Comment:
Remove extra line.

Reply:
I trust your judgement but I did not test it w/o those parameters.
Particularly I see it in the docs
| Name | Description | Type | Default | Required | Notes | Tasks |
| :--- | ---: | ---: | ---: | ---: | ---: | ---: |
..........................
| V4_CFG_CADENCE_NAME | Cadence name | string | **lts** | false | [stable,lts] | viya |
| V4_CFG_CADENCE_VERSION | Cadence version | string | **2020.1** | **true** | | viya |

Reply:
Just asking here to remove line number 23 that's all. ;)